### DISCUSSION TITLE
Tooltip invokerRelation label and aria-hidden

### DISCUSSION THREAD

**Discussion by vlaraort** [ing-bank/lion#2016] [General]
👋

I am using the tooltip component in a lion-button, without text, and using the invokerRelation label, as it's documented:
https://lion-web.netlify.app/components/tooltip/use-cases/

Testing in screen readers, I noticed that the label, is read twice, one in the button, and another one in the tooltip.

As the behaviour of the invokerRetation label is to create a labelledBy, always the text of the invoker and the tooltip is going to be the same, so IMO a good behaviour for screen reader users will be to add a aria-hidden to the tooltip, hidding it totally to SR, and  avoiding duplicated information, as the invoker will already provide the label to the user.

WDYT?

**Comment by gerjanvangeest** [comment]
This is a duplicate of: https://github.com/ing-bank/lion/issues/2014
Let's continue the discussion there :)

**Reply by vlaraort** [reply]
Sorry @gerjanvangeest but it's not a duplicate, it's another behaviour. #2014 refers to hide the arrow of the tooltip. This discussion is about adding aria-hidden to the complete overlay content, when the invokerRelation is set to "label", so for a screen reader user, there will be not overlay at all, and the invoker will be labelled by the content of the hidden overlay.

**Reply by gerjanvangeest** [reply]
Ah sorry my bad, i thought you meant the same thing.

I quickly tested our [tooltip demo](https://lion-web.netlify.app/components/tooltip/use-cases/) with VoiceOver, and there it only said the label once. How did you test it to get it announced twice?

**Reply by vlaraort** [reply]
I just tested in this case, and I read it twice:

Agenda, button (ctrl-opt-right)
Agenda, tooltip (ctrl-opt-right)
(Out of the element)

It's not your case? I tested voiceover on chrome
